source/s3_auto_ranged_put.c

```diff
@@ -528,7 +427,7 @@ static int s_s3_auto_ranged_put_header_block_done(
     struct aws_allocator *allocator = meta_request->allocator;
     AWS_PRECONDITION(allocator);
 
-    if (request->desc_data.request_tag == AWS_S3_AUTO_RANGED_PUT_REQUEST_TAG_CREATE_MULTIPART_UPLOAD) {
+    if (request->request_tag == AWS_S3_AUTO_RANGED_PUT_REQUEST_TAG_CREATE_MULTIPART_UPLOAD) {
```

Comment:
Oh. Is request_tag really just state machine state/process phase/stage? Should we indicate that in its name/documentation?

Reply:
Not really state--it's an enum that is defined by the derived class type that indicates the request type.  (Like create-multipart-upload for example.)  I wanted to allow them to include context where that's convenient (like *first*_ranged_get) so they don't map 1:1, which is why the aren't a global "s3 request types" enum.

Reply:
Changed the documentation a little bit to be more clear.